src/ng-backstretch.js

```diff
@@ -11,9 +11,9 @@ directive('backstretch', ['$window', '$timeout', function($window, $timeout) {
   return {
     restrict: 'A',
     scope: {
-      images: '&backstretchImages',
-      duration: '&backstretchDuration',
-      fade: '&backstretchFade'
+      images: '=backstretchImages',
```

Comment:
Is there a reason why we need two-way binding for these attributes? I try to limit scope exposure to reduce process overhead — even as little as this may be.